xline/src/main.rs

```diff
@@ -201,6 +201,9 @@ struct ServerArgs {
     /// Curp client retry timeout
     #[clap(long, value_parser = parse_duration)]
     client_retry_timeout: Option<Duration>,
+    /// Range request retry timeout
```

Comment:
Please add the default value of this `range_retry_timeout` in the comment.

Reply:
Added